src/harness/testcases/WINNF_FT_S_PAT_testcase.py

```diff
@@ -104,10 +103,12 @@ def test_WINNF_FT_S_PAT_1(self, config_filename):
     
     config = loadConfig(config_filename)
     num_tests = len(config)
-    test_pass_threshold = 1.00
-    passed_tests = 0.0
-    for configItem in  config:
-       request= config[configItem]
+    test_pass_threshold = 0.95 #percentage of passed test expected
+    num_passed_tests = 0.0
+    num_tests = len(config)
+    max_fail_num = int ((1-test_pass_threshold) * num_tests)
```

Comment:
You probably want to explicitly use `ceiling` here.

Reply:
Say num_tests = 81
test_pass_threshold = 90%
(1-test_pass_threshold) * num_tests = 8.1
You want the max_fail_num to be 8 and not 9.

Reply:
But if I fail 8, I've passed 81-8 tests and (81-8)/81 = 90.1% which is greater than 90%. So even if I fail 8 tests, I may still pass the overall test case. It's only once I fail the 9th test case that I've definitely failed the overall test.

Reply:
Oops, sorry, you're right -- the way you've defined and used the variable, floor() makes more sense.

Reply:
Please use `floor` explicitly.

Reply:
Please either address this comment or say why you disagree with it.

Reply:
> The maximum number of failed test is
> max_fail_num = int ((1-test_pass_threshold) * num_tests)
> and the check is
> num_failed_tests > max_fail_num
> because
> num_passed_tests >= num_tests * test_pass_threshold (at least 99.9% of trials)

I agree that the logic is correct and that floor is the right behavior. My suggestion was to use the floor function directly --
 rather than relying on truncation during the cast to integer, and also to improve readability. A comment would also work.